.github/workflows/container-image.yml

```diff
@@ -47,6 +47,31 @@ jobs:
       with:
         image: "samba-server:latest"
         retention_days: 1
+  build-server-centos:
+    runs-on: ubuntu-latest
+      #env:
+      #      BUILDAH_FORMAT: oci
+    steps:
+      -  uses: actions/checkout@v3
+      -   name: Build the CentOS based server image
+          run: make OS_NAME=centos build-server
+    - name: Upload server image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "samba-server:centos-latest"
+        retention_days: 1
+
+  test-centos-server:
```

Comment:
`test-server-centos` would be better to match the naming pattern used elsewhere in the file.

Reply:
I did that yesterday. 
@anoopcs9  wrote:
> `test-server-centos` would be better to match the naming pattern used elsewhere in the file.

Good point! I did that yesterday.